openerp/addons/openupgrade_records/lib/apriori.py

```diff
@@ -29,6 +29,7 @@
     'marketing_security_group': 'mass_mailing_security_group',
     # odoomrp/odoomrp-wip
     'product_variants_no_automatic_creation': 'product_variant_configurator',
+    'sale_documents_comments': 'sale_comment_propagation',
```

Comment:
the module https://github.com/OCA/sale-workflow/pull/509 is still named `sale_comment_propagation`, see last commit